app/views/gems_adoptions/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, "Aditional Information") %>
+<h1><%= @gem_adoption %></h1>
+<%= form_for @gem do |g| %>
```

Comment:
It would be nice to display the errors messages here in case there was an error while putting creating the GemsAdoption. One simple way to display the errors here is:

```ruby
<% if @gem.errors.any? %>
      <div>
        <h2>
          <%= pluralize(@gem.errors.count, "error") %> prohibited
          this article from being saved:
        </h2>
        <ul>
          <% @gem.errors.full_messages.each do |msg| %>
            <li><%= msg %></li>
          <% end %>
        </ul>
      </div>
    <% end %>
```ruby

@angela4713 let us know if you need a bit of help to do this.